docs/sdk/js/cross-contract/callbacks.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 2
+title: Callbacks
+---
+
+# Callbacks
+
+NEAR Protocol is a sharded, proof-of-stake blockchain that behaves differently than proof-of-work blockchains. When interacting with a native Rust (compiled to Wasm) smart contract, cross-contract calls are asynchronous. Callbacks are used to either get the result of a cross-contract call or tell if a cross-contract call has succeeded or failed.
+
+## Calculator Example
+
+A callback method can be declared in your contract class as a regular method decorated with the `call({})` decorator. Be sure to pass in the `privateFunction: true` option to the decorator. This will ensure that the method is only callable by the contract itself.
+
+For example, let's assume the calculator is deployed on `calc.near`, we can use the following:
+
+```js
+@NearBindgen({})
+export class CalculatorCallerContract {
+  @call({})
```

Comment:
In the paragraph above, you mention `Be sure to pass in the 'privateFunction: true' option`, but I don't see the option in these code examples? 🤔

Reply:
The callback method is `sum` belonging to the `calculatorAccountId` account. That method should have the decorator param. But let me add that here because I understand this is confusing.